wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandScheduler.java

```diff
@@ -266,17 +267,20 @@ public void run() {
     m_watchdog.addEpoch("buttons.run()");
 
     m_inRunLoop = true;
+    boolean isDisabled = RobotState.isDisabled();
     // Run scheduled commands, remove finished commands.
     for (Iterator<Command> iterator = m_scheduledCommands.iterator(); iterator.hasNext(); ) {
       Command command = iterator.next();
 
-      if (!command.runsWhenDisabled() && RobotState.isDisabled()) {
+      if (!command.runsWhenDisabled() && isDisabled) {
```

Comment:
Nitpick: if `isDisabled` is a variable, perhaps flip the operands here to short-circuit away the runsWhenDisabled call?